munit/shared/src/main/scala/munit/MUnitRunner.scala

```diff
@@ -310,6 +310,9 @@ class MUnitRunner(val cls: Class[_ <: Suite], newInstance: () => Suite)
         handleNonFatalOrStackOverflow(ex)
       case ex: StackOverflowError =>
         handleNonFatalOrStackOverflow(ex)
+      case ex =>
```

Comment:
If this is a fatal exception, would it make sense to abort the suite then, or not really/it doesn't matter? :thinking:

Reply:
Very good idea 👍